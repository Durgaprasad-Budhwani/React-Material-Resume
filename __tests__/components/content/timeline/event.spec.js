/**
 * Created by durgaprasad on 10/10/17.
 */
/* global expect,it */
import React from 'react';
import Event from '../../../../src/components/content/timeline/event';
import renderer from 'react-test-renderer';

it('renders event', () => {
  const tree = renderer.create(
    <Event
      year="name1"
      company="abc"
      title="abc"
      description="red"
      alignment={'LEFT'}
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});