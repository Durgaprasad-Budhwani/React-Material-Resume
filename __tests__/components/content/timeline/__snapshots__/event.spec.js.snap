// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders event 1`] = `
<div
  align="center"
  className="MuiGrid-typeContainer-5 MuiGrid-spacing-xs-16-26 MuiGrid-direction-xs-column-7"
>
  <div
    className="Event-date-1"
  >
    name1
  </div>
  <h3
    className="Event-header3-2"
  >
    abc
  </h3>
  <h4
    className="Event-header4-3"
  >
    abc
  </h4>
  <p
    className="Event-description-4"
  >
    red
  </p>
</div>
`;
